src/wallet/rpc/spend.cpp

```diff
@@ -1159,6 +1160,267 @@ RPCHelpMan send()
     };
 }
 
+RPCHelpMan sweep()
+{
+    return RPCHelpMan{"sweep",
+        "\nEXPERIMENTAL warning: this call may be changed in future releases.\n"
+        "\nSpend all confirmed UTXOs in the wallet to one or more recipients.\n"
+        "Unconfirmed inbound UTXOs and locked UTXOs will not be spent. Sweep will respect the wallet flag for avoid_reuse.\n",
+        {
+            {"receivers", RPCArg::Type::ARR, RPCArg::Optional::NO, "The destinations of the sweep, each address may only appear once.\n"
+                "Optionally some receivers can be specified with an amount, but at least one address must appear without a specified amount.\n",
+                {
+                    {"address", RPCArg::Type::STR, RPCArg::Optional::NO, "A bitcoin address which receives an equal share of the unspecified amount."},
+                    {"", RPCArg::Type::OBJ_USER_KEYS, RPCArg::Optional::OMITTED, "",
+                        {
+                            {"address", RPCArg::Type::AMOUNT, RPCArg::Optional::NO, "A key-value pair. The key (string) is the bitcoin address, the value (float or string) is the amount in " + CURRENCY_UNIT + ""},
```

Comment:
nit:
I think it's better to replace `key-value` -> `key:value`, because that's how the pair is originally expressed in arguments.

Reply:
"key-value" is the prevalent way of describing this sort of datum.

Reply:
> "key-value" is the prevalent way of describing this sort of datum.

I had checked with various RPC commands, and I think it is correct.